habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/ProcessExecutor.java

```diff
@@ -76,7 +76,6 @@ public String executeAndGetResult(final Logger logger) {
         }
     }
 
-    @SuppressWarnings("deprecation")
```

Comment:
I: This is the only real change and it was an unnecessary suppression. The rest is just formatting.